pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "pyansys"
+version = "2023.1.dev0"

```

Comment:
```suggestion
version = "2022.1.dev0"

```

Reply:
I would leave it as it is. The current "dev" version is intended to target 2023R1.

Reply:
Agreed. Resolving...